core/src/main/java/bio/terra/pearl/core/dao/participant/FamilyDao.java

```diff
@@ -22,4 +24,11 @@ protected Class<Family> getClazz() {
     public Optional<Family> findOneByShortcode(String shortcode) {
         return findByProperty("shortcode", shortcode);
     }
+
+    public List<Family> findByEnrolleeId(UUID enrolleeId) {
+        return jdbi.withHandle(handle -> handle.createQuery("SELECT f.* FROM family f INNER JOIN family_member fm ON fm.family_id = f.id WHERE fm.enrollee_id = :enrolleeId")
```

Comment:
`f` and `fm` aren't great distinguishers.  I would either not abbreviate, or just use "family" and "member"